ports/servoshell/desktop/headed_window.rs

```diff
@@ -139,6 +139,8 @@ impl Window {
         let rendering_context =
             Rc::new(window_rendering_context.offscreen_context(rendering_context_size));
 
+        winit_window.set_ime_allowed(true);
```

Comment:
This doesn't work on Linux and Windows. I suppose you need to implement app_state::show/hide_ime in some way.

Reply:
Okay I'll look into it.